Source/utils/endian.hpp

```diff
@@ -4,6 +4,14 @@
 
 namespace devilution {
 
+template <typename T>
+constexpr std::uint16_t LoadLE16(const T *b)

```

Comment:
Could I request a `LoadBE16` be added and used for in `NetRecvPlrData`?

Reply:
Sure but that's for a separate PR